examples/simplecell/simplecell.ipynb

```diff
@@ -0,0 +1,861 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pipeline.store_optimisation_results()\n",
+    "pipeline.validation()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The `./final.json` contains the best model across all seeds. Since we're only running a single seed here, the result will simply be that model. Now, let's check the final results"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{\n",
+      "    \"simplecell__1\": {\n",
+      "        \"emodel\": \"simplecell\",\n",
+      "        \"etype\": \"cADpyr\",\n",
+      "        \"ttype\": null,\n",
+      "        \"mtype\": null,\n",
+      "        \"species\": \"rat\",\n",
+      "        \"brain_region\": \"SSCX\",\n",
+      "        \"iteration\": null,\n",
+      "        \"synapse_class\": null,\n",
+      "        \"allen_notation\": null,\n",
+      "        \"score\": 510.2861723258447,\n",
+      "        \"parameters\": {\n",
+      "            \"gnabar_hh.somatic\": 0.05000000003749999,\n",
+      "            \"gkbar_hh.somatic\": 0.07499999996750001\n",
+      "        },\n",
+      "        \"fitness\": {\n",
+      "            \"IDrest_0.2.soma.v.Spikecount\": 1.1094003924504583,\n",
+      "            \"IDrest_0.2.soma.v.mean_frequency\": 250.0,\n",
+      "            \"IDrest_0.2.soma.v.voltage_base\": 6.86265875272435,\n",
+      "            \"IV_-0.1.soma.v.voltage_base\": 2.3141131806698945,\n",
+      "            \"IV_-0.1.soma.v.ohmic_input_resistance_vb_ssse\": 250.0\n",
+      "        },\n",
+      "        \"features\": {\n",
+      "            \"IDrest_0.2.soma.v.Spikecount\": 1.0,\n",
+      "            \"IDrest_0.2.soma.v.mean_frequency\": 1739.1304358817733,\n",
+      "            \"IDrest_0.2.soma.v.voltage_base\": -82.86184220245485,\n",
+      "            \"IDrest_0.4.soma.v.Spikecount\": 1.0,\n",
+      "            \"IDrest_0.4.soma.v.mean_frequency\": 2105.263159505021,\n",
+      "            \"IDrest_0.4.soma.v.voltage_base\": -82.71053576131624,\n",
+      "            \"IV_-0.1.soma.v.voltage_base\": -84.28296205269712,\n",
+      "            \"IV_-0.1.soma.v.ohmic_input_resistance_vb_ssse\": 1060.243113329033\n",
+      "        },\n",
+      "        \"validation_fitness\": {\n",
+      "            \"IDrest_0.4.soma.v.Spikecount\": 250.0,\n",
+      "            \"IDrest_0.4.soma.v.mean_frequency\": 250.0,\n",
+      "            \"IDrest_0.4.soma.v.voltage_base\": 250.0\n",
+      "        },\n",
+      "        \"validated\": false,\n",
+      "        \"seed\": 1,\n",
+      "        \"pdfs\": [\n",
+      "            \"/gpfs/bbp.cscs.ch/home/ikilic/workspace/BluePyEModel/examples/simplecell/figures/simplecell/optimisation/emodel=simplecell__etype=cADpyr__species=rat__brain_region=SSCX__seed=1__optimisation.pdf\",\n",
```

Comment:
Can also remove /gpfs paths here after removing them from final.json

Reply:
Removed in the last commit